### ISSUE TITLE
Add the ability to switch between acrylic and explorer blur on windows 10

### ISSUE THREAD

**Issue by windowsxp3790** [Maplespe/DWMBlurGlass#51]
In a discord server we did some testing and found that acrylic can perform better in some ways then explorer blur on windows 10
Other than acrylic only updating at 30fps or whatever it's better becuse at least it doesn't progressively slow down your window movement the more windows you have open
Only having 6/7 windows open is enough for explorerblur to already become slow as hell for just dragging things around while acrylic is locked at 30fps no matter how many windows you have open
So basically acrylic is just more consistent in performance

**Comment by Maplespe** [comment]
As I've said many times, Acrylic simply doesn't work, not only does it have no effect in 22H2 but the drag delay is 100x slower than Blur.
But where you do a search you get tons of reports of this problem.
[https://www.google.com/search?q=windows+acrylic+lag](https://www.google.com/search?q=windows+acrylic+lag)

[https://github.com/tauri-apps/window-vibrancy/issues/47](https://github.com/tauri-apps/window-vibrancy/issues/47) Windows Acrylic effect lag

[https://github.com/Seo-Rii/electron-acrylic-window/issues/6](https://github.com/Seo-Rii/electron-acrylic-window/issues/6) lag

[https://github.com/EYHN/vscode-vibrancy/issues/5](https://github.com/EYHN/vscode-vibrancy/issues/5) Problem drag, move window by Title Bar

This is my last reply to this issue, here is a video of the actual test:

[Desktop 2024.01.26.mp4.zip](https://github.com/Maplespe/DWMBlurGlass/files/14059736/Desktop.2024.01.26.mp4.zip)

We're already working on custom blur, and there's no use in rushing.

**Comment by TheInsane101** [comment]
> We're already working on custom blur, and there's no use in rushing.
 
Does that you mean you are developing a new shader like Big Muscle did with Glass8? If so, that is absolutely amazing! If not, that's fine too!

**Comment by Maplespe** [comment]
> Does that you mean you are developing a new shader like Big Muscle did with Glass8? If so, that is absolutely amazing! If not, that's fine too!

Yes, not only that, but it's no exaggeration to say that it's also more powerful than glass8's effect, which will be unprecedented.

**Comment by ALTaleX531** [comment]
Not a shader, to exact, but our own implementation of the blurring method😊, which is still very different from Glass 8 ( btw: Glass 8 uses not a shader, but a reproduction of the Windows 7 blurring mechanism).
Here's a preview of what you might be interested in.😎
![image](https://github.com/Maplespe/DWMBlurGlass/assets/105535231/73b48c0d-3b10-4145-b82b-3960d089ba98)

**Comment by TheInsane101** [comment]
> Not a shader, to exact, but our own implementation of the blurring method😊, which is still very different from Glass 8 ( btw: Glass 8 uses not a shader, but a reproduction of the Windows 7 blurring mechanism). Here's a preview of what you might be interested in.😎 ![image](https://private-user-images.githubusercontent.com/105535231/302094102-73b48c0d-3b10-4145-b82b-3960d089ba98.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcwNDY0MjAsIm5iZiI6MTcwNzA0NjEyMCwicGF0aCI6Ii8xMDU1MzUyMzEvMzAyMDk0MTAyLTczYjQ4YzBkLTNiMTAtNDE0NS1iODJiLTM5NjBkMDg5YmE5OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwNFQxMTI4NDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mMjJjYWJhOGJjNWNiNTc4Y2I3M2QzYmJhZTMxNTUzMTQ2ZjllOTNiNDk1ZGY5MDk0OTBjNWM1ZmIwNDQ5OTExJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.YHty6WVi3HLav9_SVDk1XBj6xRxm-nbMyDPU_xWTz5w)

Tbh I don't see the difference compared to the current version (I don't use Windows 11 so I don't know what the current 11 version looks like) but I like the shadow borders. The ability to extend the effect into entire windows looks so cool though! Excited! Btw you're being quite nice; I appreciate it 😊

Will the reflection also be layered on top of the colour layer (so it's more visible) and have its movement mimic Windows 7's more accurately?

**Comment by Maplespe** [comment]
This image should be more obvious:
The Aero reflection is at the very top, and the inactive window also has the exposure and saturation effects of Aero (which is the real Aero effect).
![195131](https://github.com/Maplespe/DWMBlurGlass/assets/43676879/c35abe15-8fee-4c1d-b022-739dd13106c1)

**Comment by TheInsane101** [comment]
> This image should be more obvious: The Aero reflection is at the very top, and the inactive window also has the exposure and saturation effects of Aero (which is the real Aero effect). ![195131](https://private-user-images.githubusercontent.com/43676879/302100868-c35abe15-8fee-4c1d-b022-739dd13106c1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcxMTY3NzAsIm5iZiI6MTcwNzExNjQ3MCwicGF0aCI6Ii80MzY3Njg3OS8zMDIxMDA4NjgtYzM1YWJlMTUtOGZlZS00YzFkLWIwMjItNzM5ZGQxMzEwNmMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA1VDA3MDExMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZmYjYyNjlmZjdjM2RlZjlhNzRlMjExMjQzZWFhZGQ4ZjIxZjE0Y2Q1ZDJiMWE0ZDRkZjAzMGMyNWQ0NDAyMWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.WQ5sjQZjUkwH6u56CrvfdeHTqQEiVivgmMtlQc0EZBM)

Oh now I can see it much better. It looks **fantastic** and exactly like Windows 7's Aero! I wish you so much luck with this!😍

**Comment by Maplespe** [comment]
Updated in the new version.
